x-pack/packages/security-solution/upselling/messages/index.tsx

```diff
@@ -48,8 +48,8 @@ export const ALERT_SUPPRESSION_RULE_DETAILS = i18n.translate(
 );
 
 export const UPGRADE_NOTES_MANAGEMENT_USER_FILTER = (requiredLicense: string) =>
-  i18n.translate('securitySolutionPackages.noteManagement.userFilter.upsell', {
-    defaultMessage: 'Upgrade to {requiredLicense} to make use of user filters',
+  i18n.translate('securitySolutionPackages.noteManagement.createdByFilter.upsell', {
+    defaultMessage: 'Upgrade to {requiredLicense} to make use of createdBy filter',
```

Comment:
what do you mean by their server side?

Reply:
Corrected my statement above. Sorry. I meant if we check on server as well for the user license.. to make sure that user with basic license are not able to execute this feature.

Reply:
this change is just a translation text change. In a previous [PR](https://github.com/elastic/kibana/pull/197149) I added some checks in the frontend to disable the combobox on those licenses, because the page was breaking because of the api failing

Reply:
ok now I see, let me take a look at that tomorrow, good point!

Reply:
so I looked into it more and here's what I found:
- the [suggestedUserProfileRoute](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/server/lib/detection_engine/routes/users/suggest_user_profiles_route.ts#L47) that is used to populate the dropdown in the notes management page here (as well as the alert assign button on our alerts page) uses [this suggest method](https://github.com/elastic/kibana/blob/main/packages/core/user-profile/core-user-profile-server/src/service.ts#L55) which has a `requiredPrivileges` property. For my [getNotes](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/server/lib/timeline/routes/notes/get_notes.ts#L30) endpoint I am using the [bulkGet](https://github.com/elastic/kibana/blob/main/packages/core/user-profile/core-user-profile-server/src/service.ts#L43) method, which does not have that. I'm not sure why...
- I'm not sure this is actually an issue here. There is no license or privilege limitation that would prevent a user from seeing other users' notes. That is the whole point of notes, we want users to collaborate. Because the logic is all self contained on the server and no user information leaks out, I don't think we're having an issue here. I guess the only thing is we prevent the user from seeing the dropdown on the notes management page. I don't really know why this is necessary...

Do you agree with the above? Do you think we still have an issue here?

Reply:
So do you mean, we just upsell the UI for this feature and not the API?

I think this is the product question. I just wanted to bring it to your attention. It feels strange to just upsell the UI but not unheard of. Maybe just confirm with Paul after this PR?

Reply:
also checking in other places in the code, it seems that cases is using the same `bulkGet` method and it's not protected behind a license paywall or anything.
I believe the only issue here is the inconsistency between the UI and the API. Hopefully not a big deal to Paul's eyes 😆 🤞